content/en/docs/tasks/debug/debug-cluster/flow-control.md

```diff
@@ -0,0 +1,178 @@
+---
+title: Troubleshooting APIServer flow control issues
+content_type: task
+---
+
+<!-- overview -->
+
+API Priority and Fairness controls the behavior of the Kubernetes API server in
+an overload situation. You can find more information about it in the
+[concept doc](/docs/concepts/cluster-administration/flow-control.md)
+
+## {{% heading "prerequisites" %}}
+
+* Kubernetes cluster is installed
+* `kubectl` is configured to communicate with the cluster
+
+<!-- steps -->
+
+## Diagnostics
+
+Every HTTP response from an API server with the priority and fairness feature
+enabled has two extra headers: `X-Kubernetes-PF-FlowSchema-UID` and
+`X-Kubernetes-PF-PriorityLevel-UID`, noting the flow schema that matched the request
+and the priority level to which it was assigned, respectively. The API objects'
+names are not included in these headers in case the requesting user does not
+have permission to view them, so when debugging you can use a command like
+
+```shell
+kubectl get flowschemas -o custom-columns="uid:{metadata.uid},name:{metadata.name}"
+kubectl get prioritylevelconfigurations -o custom-columns="uid:{metadata.uid},name:{metadata.name}"
+```
+
+to get a mapping of UIDs to names for both FlowSchemas and
+PriorityLevelConfigurations.
```

Comment:
Also consider:
```shell
kubectl get flowschemas -o jsonpath='{range .items[?(.metadata.uid=262bab1a-1c79-11ea-8e23-42010a800016)]}{.metadata.name}{end}'
```
or similar

Reply:
I'm not a jsonpath expert. But this doesn't seems to be right to output a list of <name, uid> pairs.